doc/source/WorkingWithImages/AnatomyOfAnImage/screengrab.py

```diff
@@ -0,0 +1,86 @@
+script["Outputs"]["outputs"]["output1"]["parameters"]["displayHost"].addChild( Gaffer.StringPlug( "value", defaultValue = 'localhost', flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+script["Outputs"]["outputs"]["output1"]["parameters"].addChild( Gaffer.CompoundDataPlug.MemberPlug( "remoteDisplayType", flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+script["Outputs"]["outputs"]["output1"]["parameters"]["remoteDisplayType"].addChild( Gaffer.StringPlug( "name", defaultValue = 'remoteDisplayType', flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+script["Outputs"]["outputs"]["output1"]["parameters"]["remoteDisplayType"].addChild( Gaffer.StringPlug( "value", defaultValue = 'GafferImage::GafferDisplayDriver', flags = Gaffer.Plug.Flags.Default | Gaffer.Plug.Flags.Dynamic, ) )
+script["SceneReader"]["fileName"].setValue( '${GAFFER_ROOT}/resources/gafferBot/caches/gafferBot.scc' )
+script["Camera"]["transform"]["translate"].setValue( imath.V3f( -10, 13, 40 ) )
+script["Group"]["in"]["in0"].setInput( script["SceneReader"]["out"] )
+script["Group"]["in"]["in1"].setInput( script["Camera"]["out"] )
+script["StandardOptions"]["in"].setInput( script["Group"]["out"] )
+script["StandardOptions"]["options"]["renderCamera"]["value"].setValue( '/group/camera' )
+script["StandardOptions"]["options"]["renderCamera"]["enabled"].setValue( True )
+script["InteractiveAppleseedRender"]["in"].setInput( script["Outputs"]["out"] )
+script["Catalogue"]["directory"].setValue( '${project:rootDirectory}/catalogues/${script:name}' )
+script["Outputs"]["in"].setInput( script["StandardOptions"]["out"] )
+script["Outputs"]["outputs"]["output1"]["name"].setValue( 'Interactive/Beauty' )
+script["Outputs"]["outputs"]["output1"]["fileName"].setValue( 'beauty' )
+script["Outputs"]["outputs"]["output1"]["type"].setValue( 'ieDisplay' )
+script["Outputs"]["outputs"]["output1"]["data"].setValue( 'rgba' )
+
+def __renderAndGrab( script, widget, imagePath, delay = 15 ) :
+	script["variables"]["imageCataloguePort"]["value"].setValue( script["Catalogue"].displayDriverServer().portNumber() )
+	script["InteractiveAppleseedRender"]["state"].setValue( script["InteractiveAppleseedRender"].State.Running )
+	__delay( delay )
+
+	viewport = scriptWindow.getLayout().editors( GafferUI.Viewer )[0].view().viewportGadget()
+	viewport.frame( viewport.getPrimaryChild().bound() )
+	GafferUI.EventLoop.waitForIdle()
+
+	GafferUI.WidgetAlgo.grab( widget = widget, imagePath = imagePath )
+	script["InteractiveAppleseedRender"]["state"].setValue( script["InteractiveAppleseedRender"].State.Stopped )
+
+script.selection().add( script["Catalogue"] )
+# Dark magic code, to be fixed later
+__delay( 1 )
+__toolbar = viewer.viewGadgetWidget()._GLWidget__graphicsScene._GLGraphicsScene__overlays.items()[0][0][0][0].getChild()
```

Comment:
A super-hacky way of displaying the channel menu. @johnhaddon Let me know if there's a better way.

Reply:
Can I suggest that we just drop this particular screengrab for now? I'm getting this error intermittently here :

```
File "screengrab.py", line 85, in <module>
    __menu = __toolbar._StandardNodeToolbar__layout._qtWidget().children()[1].children()[1].children()[3].children()[1]
AttributeError: 'ListContainer' object has no attribute '_StandardNodeToolbar__layout'
```

Furthermore, that particular menu is a bit misleading to show in this case, because it's not really showing the channel names - instead it's showing the 0-4th channels for the current layer.